ru/ru_barcode_targets.py

```diff
@@ -106,6 +106,14 @@
             default=None,
         ),
     ),
+    (
+        "--odd-even",
+        dict(
+            default=False,
```

Comment:
This is a nitpick, but the default is unnecessary here.

Reply:
I know your IP address